arunananth.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login page</title>
+</head>
+<body>
+    <h2>Login</h2>
+    <input type="text" placeholder="Enter user name">
+    <input type="password" placeholder="Enter password">
+    <button type="submit">Submit</button>
```

Comment:
why you write type="submit"

Reply:
this is summit button. so, i give the type in submit.

Reply:
ok